ui/client/components/graph/node-modal/editors/expression/EditorType.ts

```diff
@@ -48,20 +70,41 @@ export const editors: Record<editorTypes, Editor> = {
     editor: () => FixedValuesEditor,
     hint: (switchable) => switchable ? FixedValuesEditor.switchableToHint : FixedValuesEditor.notSwitchableToHint,
     showSwitch: false,
-    switchableTo: (expressionObj, param, values) => FixedValuesEditor.switchableTo(expressionObj, !_.isEmpty(values) ? values : param.editor.possibleValues),
+    switchableTo: (expressionObj, param, values) => FixedValuesEditor.switchableTo(
+        expressionObj,
+        !_.isEmpty(values) ? values : param.editor.possibleValues,
+    ),
     values: (param, values) => !_.isEmpty(values) ? values : param.editor.possibleValues,
   },
   [editorTypes.DUAL_PARAMETER_EDITOR]: {
     editor: (param, displayRawEditor) => displayRawEditor ?
-      editors[editorTypes.RAW_PARAMETER_EDITOR].editor() :
-      editors[param.editor.simpleEditor.type].editor(),
+        editors[editorTypes.RAW_PARAMETER_EDITOR].editor() :
+        editors[param.editor.simpleEditor.type].editor(),
     hint: (switchable, currentEditor, param) => currentEditor === RawEditor ?
-      editors[param.editor.simpleEditor.type].hint(switchable) :
-      editors[editorTypes.RAW_PARAMETER_EDITOR].hint(),
+        editors[param.editor.simpleEditor.type].hint(switchable) :
+        editors[editorTypes.RAW_PARAMETER_EDITOR].hint(),
     showSwitch: true,
     switchable: (editor, param, expressionObj) => editor === RawEditor ?
-      editors[param.editor.simpleEditor.type].switchableTo(expressionObj) :
-      true,
+        editors[param.editor.simpleEditor.type].switchableTo(expressionObj) :
+        true,
     values: (param) => param.editor.simpleEditor.possibleValues,
   },
+  [editorTypes.DATE]: {
+    editor: () => DateEditor,
+    hint: switchable => switchable ? DateEditor.switchableToHint : DateEditor.notSwitchableToHint,
+    showSwitch: true,
```

Comment:
It should be set to false -> showSwitch should be true only ion dual editor -> we do not want to show switch in other editors